kyo-combinators/shared/src/main/scala/kyo/AsyncCombinators.scala

```diff
@@ -104,14 +100,14 @@ extension [A, E, S](fiber: Fiber[E, A] < S)
       * @return
       *   A computation that produces the result of this computation with Async effect
       */
-    def join(using Frame): A < (S & Abort[E] & Async) =
+    def join(using frame: Frame, reduce: Reducible[Abort[E]]): A < (S & reduce.SReduced & Async) =
```

Comment:
Is it still useful `reduce: Reducible[Abort[E]]`, with `Async <: Abort[Nothing]`?